web/src/components/CurrentSong.js

```diff
@@ -47,15 +54,32 @@ function CurrentSong({song, dj, startTime, offset}) {
           <span className="dj-name">{dj.name}</span>
         </div>
         <div className="skip">
-          <button className="btn" disabled>
+          <button className="btn" onClick={doVoteSkip}>
             Skip

```

Comment:
wdyt about the idea of making this button say "Unskip" if the user has voteskipped, and also keeping the vote count gray ("inactive") and only making it active if the user has voteskipped?